api/Dockerfile

```diff
@@ -23,6 +23,4 @@ COPY --from=builder ${VIRTUAL_ENV} ${VIRTUAL_ENV}
 
 COPY . ./
 
-RUN manage.py collectstatic
-
```

Comment:
**question:** why?

Reply:
i get error message because of this: 
![image](https://github.com/knmlprz/ChatKNML/assets/146839542/0bc0be51-e848-4751-90d0-f799d379dc8b)
i'm propably wrong but in my opinion command should looks like: " RUN python manage.py collectstatic", but then i get another error, because (i think) i can't get data contained in .env file
![image](https://github.com/knmlprz/ChatKNML/assets/146839542/e7c2e277-6b55-4487-a78a-318db3a2e8c1)

so i cant currently use this command and temporarily deleted them

if you can, correct me if i'am wrong

Reply:
it doesn't work, because
1. you didn't change workdir in the target stage
2. you have to use relative path, unlike on windows, linux treats `SOME_COMMAND` as thing present in PATH and current working directory IS NOT automatically added to path, hence you have to use `./manage.py` instead